src/odemis/util/dataio.py

```diff
@@ -221,7 +221,7 @@ def open_acquisition(filename, fmt=None):
     return data
 
 
-def splitext(path):
+def splitext(path):  # TODO should that not be "splittext"?
```

Comment:
No, it's supposed to be a slightly improved version of `os.path.splitext()`. Which is (probably the abbreviation of "split extension")